qt/scientific_interfaces/ISISReflectometry/GUI/Runs/RunsPresenter.cpp

```diff
@@ -239,22 +227,19 @@ void ReflRunsTabPresenter::populateSearch(IAlgorithm_sptr searchAlg) {
 /** Searches ICAT for runs with given instrument and investigation id, transfers
  * runs to table and processes them. Clears any existing table data first.
  */
-void ReflRunsTabPresenter::startNewAutoreduction() {
+void RunsPresenter::startNewAutoreduction() {
   if (requireNewAutoreduction()) {
     // If starting a brand new autoreduction, delete all rows / groups in
     // existing table first
     // We'll prompt the user to check it's ok to delete existing rows
 
-    // TODO: enable autoprocessing
-    //   auto tablePresenter = getTablePresenter(group);
-    //   tablePresenter->setPromptUser(false);
-    //   try {
-    //     tablePresenter->notify(DataProcessorPresenter::DeleteAllFlag);
-    //   } catch (const DataProcessorPresenter::DeleteAllRowsCancelledException
-    //   &)
-    //   {
-    //     return;
-    //   }
+    // tablePresenter()->setPromptUser(false);
```

Comment:
I'm going to ignore commented out code on the assumption there is plans to sort out these segments in a future PR before the feature branch is merged with master.